torchgeo/transforms/indices.py

```diff
@@ -241,3 +241,60 @@ def __init__(self, index_nir: int, index_vre1: int) -> None:
             index_vre1: index of the Red Edge band, B5 in Sentinel 2 imagery
         """
         super().__init__(index_a=index_nir, index_b=index_vre1)
+
+
+class AppendTribandNormalizedDifferenceIndex(Module):
+    """Append normalized difference index involving 3 bands as channel to image tensor.
+
```

Comment:
Can you describe the format of this in the docstring, e.g. (band_a - (band_b + band_c)) / (band_a + band_b + band_c)

Reply:
done